python/test/unit/operators/test_matmul.py

```diff
@@ -59,6 +59,7 @@
                 (128, 64, 16, 1, 4, STAGES, 1024, 1024, 1024, AT, BT, DTYPE),
                 (256, 128, 32, 1, 8, STAGES, 1024, 1024, 1024, AT, BT, DTYPE),
                 (128, 128, 32, 1, 4, STAGES, 384, 128, 640, AT, BT, DTYPE),
+                (128, 256, 64, 1, 8, STAGES, 8192, 8192, 8192, AT, BT, DTYPE),
```

Comment:
I don't understand why we need 8k here? 1k or 8k should have exactly the same failure modes but the latter would be much more expensive